### DISCUSSION TITLE
Can not login node through Web Admin

### DISCUSSION THREAD

**Discussion by Elshaffah** [raspiblitz/raspiblitz#4376] [Q&A]
After trying to set up fulcrum server on my bitcoin and lightning node, I have been unable to log into my node through the web admin; instead, it keeps displaying this:

" This site can’t be reached
http://xxx.xxx.x.xxx refused to connect.

Try:
Checking the connection
Checking the proxy and the firewall

ERR_CONNECTION_REFUSED"

My raspiblitz LAN ip address is correct, but why do i have trouble logging in to my node through the web admin on my browser? What could be the cause and how do I rectify this?

**Comment by cstenglein** [comment]
Looks like the API is not running / has problems.

**Reply by Elshaffah** [reply]
So how do I rectify the problem with the API not running?
I am thinking of downloading the FATPACK SD Card Image (Beginners - WebUI) then flash it into my SD again... hopefully this will rectify the issue
